manager/controllers/app/m4dapplication_controller.go

```diff
@@ -272,3 +299,26 @@ func (r *M4DApplicationReconciler) GetAllModules() (map[string]*app.M4DModule, e
 	}
 	return moduleMap, nil
 }
+
+// CreateAppNamespace creates a namespace in which the blueprint and the relevant resources will be running
+// It stores the generated namespace name inside app status
+func (r *M4DApplicationReconciler) CreateAppNamespace(app *app.M4DApplication) error {
+	genNamespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{GenerateName: "m4d-"}}
+	genNamespace.Labels = make(map[string]string)
+	genNamespace.Labels["ownerName"] = app.Name
+	genNamespace.Labels["ownerNamespace"] = app.Namespace
```

Comment:
I think it can be one label (too many labels make cli output ugly). Also, should we prefix it with a domain to avoid collisions? How about something like:
```
genNamespace.Labels = map[string]string{
   "m4d.ibm.com.owner": app.Namespace + "." + app.Name
}
```

Reply:
Done.